plf-packaging-resources/src/main/resources/CHANGE_LOG.txt

```diff
@@ -6,136 +6,113 @@ Changelog Platform ${org.exoplatform.platform.version}
 eXo Platform ${org.exoplatform.platform.version} 
 *************************
 
-** Tasks
-    * PLF-7042 - Upgrade Tomcat 7.0.75
-
-** Improvements
-    * PLF-6345 - Default language in Social notifications
-
 ** Bugs
-    * PLF-6326 - [ECMS Search]: Excerpt in search result contains useless information
-    * PLF-6686 - Some keys on the keyboard do not work normally when adding a comment before sharing a document
-    * PLF-6687 - intranet site name embedded in source code
-    * PLF-6724 - Performance regression caused by "TemplateService" cache settings
-    * PLF-6840 - Session timeout not working reliably
-    * PLF-6918 - Error design in attach file area when uploading a ppt, word and excel document
-    * PLF-6943 - A useless space is displayed when clicking on Show comments button in smartphone
-    * PLF-7004 - eXo platform 4.4 does not start with minimal profile
-    * PLF-7008 - [TC-PLF-4.4.0-RC01][Cluster] Stabilize Cluster under heavy load
-    * PLF-7031 - The indexing administration application doesn't work under JBoss EAP distribution
-    * PLF-7045 - Unable to preview a document in search screen
-    * PLF-7047 - Notifications window exceeds browser window frame
-    * PLF-7057 - Some user searches never end
-    * PLF-7059 - Update Gatein WCI packaged jar in PLF to 2.5.0
-    * PLF-7062 - [SOC-RDBMS]: Review the space cache usage
-    * PLF-7063 - Review the ResourceBundle cache usage
-    * PLF-7074 - Exception with infinispan tomcat clustered node
-    * PLF-7076 - [Privacy] Uploading a document in my documents fires an activity visible to my network
-    * PLF-7083 - Invalid format when saving document .docx with Microsoft Office 2016
-    * PLF-7089 - Suggestion window: replace "Inviter" with "Se connecter"
-    * PLF-7132 - Last word of the comment are displayed splitted in the Document Preview Comment
-    * PLF-7133 - Different versions of Juzu Core & Plugins in PLF Packaging Archive
-    * PLF-7147 - [web-pack add-on ] Can not edit/remove responsive containers when using the Agital template.
-    * PLF-7186 - Search Indexing Management page does not appear after PLF upgrade
-    * PLF-7187 - MixinCleanerUpgradePlugin upgrade plugin should not be activated by default on social data
-
+    * PLF-7241 - Wrong width of notification bar  
+    * PLF-7379 - Duplicated rows in jbid_io_rel table
+    * PLF-7385 - Problem displaying chat and exo toolbar in some smartphones
+    * PLF-7392 - syncAll rest service does not work correctly
+    * PLF-7407 - The synchronization is not started when calling syncAll for the second time
+    * PLF-7741 - Wrong redirection when clicking a link of a shared document
+    
+** Improvements
```

Comment:
Improvements were listed before bugs in previous versions. Any reason why you changed that ?